src/Umbraco.Core/Services/LocalizedTextServiceExtensions.cs

```diff
@@ -61,5 +61,34 @@ public static string Localize(this ILocalizedTextService manager, string key, Cu
             return variables.Select((s, i) => new { index = i.ToString(CultureInfo.InvariantCulture), value = s })
                 .ToDictionary(keyvals => keyvals.index, keyvals => keyvals.value);
         }
+
+        private static ICultureDictionary _cultureDictionary;
+
+        internal static string UmbracoDictionaryTranslate(this ILocalizedTextService manager, string text)
```

Comment:
I'm going to change this a little bit because the ICultureDictionary is based on the ILocalizationService (i.e. the Dictionary) whereas the ILoaclizedTextService is for the language files (not dictionary). This is all internal stuff which needs to be refactored along with the usage of ICultureDictionary in the future so it's no biggy but makes a tiny bit more sense.